Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>access_data and first_server_access_data are unordered lists, however from the code it appears that ordering is important. If this is the case I would suggest either using a dictionary or OrderedSet rather than relying on an List</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>You are absolutely correct, the ordering is important and I think I will migrate these to a dictionary in future. At the moment I rely on Kusto to do the ordering for me before placing the results into a list which is dumb enough to leave it in the same order as when it left Kusto.</p>

Reply:
<p>Python lists should honor ordering I think - dicts don't until Py 3.7. or 3.8</p>

Reply:
<p>They appear to honour the ordering, but I'll bear all this in mind moving forward and ensure the data structures are behaving as I assume.</p>